### PR TITLE
DHFPROD-4519: Inherit from "data-hub-module-reader" role 

### PR DESCRIPTION
**Pull request by akshaysonvane**
Removed try/catch now that data-hub-explorer-architect can access the job-info data service.

### CODE REVIEW

File: one-ui/src/main/java/com/marklogic/hub/oneui/managers/ModelManager.java [hunk 1]
```diff
@@ -182,18 +182,6 @@ public JsonNode getLatestJobInfo(String modelName) {
...
-            /*
-             * As MarkLogic server throws a FailedRequestException for both data-service module not found
-             * and if the user is unauthorized to access the module; we need to fail silently and log the
-             * issue.
-             */
-            logger.error(e.getMessage());
-        }
-        return node;
+        return JobInfo.on(dbClient).getLatestJobData(modelName);
     }
```
Comment (bsrikan) [inline one-ui/src/main/java/com/marklogic/hub/oneui/managers/ModelManager.java]: Do we have test for this in place?

Comment (akshaysonvane) [reply one-ui/src/main/java/com/marklogic/hub/oneui/managers/ModelManager.java]: Yes, the data service is tested via ml-unit-test

Comment (akshaysonvane) [reply one-ui/src/main/java/com/marklogic/hub/oneui/managers/ModelManager.java]: This PR just removes the try/catch block surrounding the code.

File: one-ui/src/main/java/com/marklogic/hub/oneui/managers/ModelManager.java [hunk 2]
```diff
@@ -182,18 +182,6 @@ public JsonNode getLatestJobInfo(String modelName) {
...
-        catch (MarkLogicServerException e) {
-            /*
-             * As MarkLogic server throws a FailedRequestException for both data-service module not found
-             * and if the user is unauthorized to access the module; we need to fail silently and log the
-             * issue.
-             */
-            logger.error(e.getMessage());
-        }
-        return node;
+        return JobInfo.on(dbClient).getLatestJobData(modelName);
```
Comment (rjrudin) [inline one-ui/src/main/java/com/marklogic/hub/oneui/managers/ModelManager.java]: I think the client should just call JobInfo then - there's no point in having this additional layer to go through. If we did keep it, I think we'd want the method names to be the same - i.e. the Manager method should be getLatestJobData.

Comment (akshaysonvane) [reply one-ui/src/main/java/com/marklogic/hub/oneui/managers/ModelManager.java]: The separate method is used for mocking the database call in the ModelManagerTest class
```
doReturn(expectedNode).when(modelManager).getJobInfoFromDB(dataServiceClient, modelName);
```

I'll change the method name to `getLatestJobData`

Comment (rjrudin) [reply one-ui/src/main/java/com/marklogic/hub/oneui/managers/ModelManager.java]: Can you mock ArtifactsService instead? That's supposed to be one of the benefits of data services - that there's an interface class that can be mocked for testing purposes.

Comment (rjrudin) [reply one-ui/src/main/java/com/marklogic/hub/oneui/managers/ModelManager.java]: But - if it's easier to just mock a single thing - i.e. ModelManager - that's fine too.

Comment (akshaysonvane) [reply one-ui/src/main/java/com/marklogic/hub/oneui/managers/ModelManager.java]: > Can you mock ArtifactsService instead? That's supposed to be one of the benefits of data services - that there's an interface class that can be mocked for testing purposes.

By ArtifactsService I am guessing you mean the JobInfo interface that was generated by the data service. 
JobInfo has a static `on` method which we call as 
```
JobInfo.on(finalDataServiceClient).getLatestJobData(s)
```
I believe mockito does not support mocking static stuff and would require us to use a testing framework like powermockito. 

Hence, in this case, it's simpler to just mock the `getLatestJobData`  method of ModelManager.

**Pull request merged**